README.md

```diff
@@ -32,10 +32,11 @@ Install dependencies
 ```sh
 python3 -m venv env
 source env/bin/activate
-pip install -r requirements-min.txt
 export FLASK_DEBUG=1
 ```
 
+Note: use `source deactivate` if you want to deactivate the virtual env
```

Comment:
Why do we need to do this with virtualenv instead of poetry?

Reply:
I think we need the virtual env for flask to work correctly if running locally